toolchain/check/import_cpp.cpp

```diff
@@ -1950,6 +1890,90 @@ static auto ImportNameDeclIntoScope(Context& context, SemIR::LocId loc_id,
                                                            access_kind);
 }
 
+// Imports an overloaded function set from Clang to Carbon.
+static auto ImportCppOverloadSet(Context& context, SemIR::NameScopeId scope_id,
+                                 SemIR::NameId name_id,
+                                 const clang::UnresolvedSet<4>& overload_set)
+    -> SemIR::InstId {
+  auto overload_set_inst = SemIR::CppOverloadSetValue{
+      SemIR::TypeId::None, SemIR::CppOverloadSetId::None};
+
+  overload_set_inst.overload_set_id = context.cpp_overload_sets().Add(
+      SemIR::CppOverloadSet{.name_id = name_id,
+                            .parent_scope_id = scope_id,
+                            .candidate_functions = overload_set});
+
+  auto overload_set_inst_id =
+      // TODO: Add a location.
+      AddPlaceholderInstInNoBlock(context, Parse::NodeId::None,
+                                  overload_set_inst);
+
+  // TODO: Check if this can be moved before adding the instruction, so that the
+  // placeholder is removed.
+  overload_set_inst.type_id = GetCppOverloadSetType(
+      context, overload_set_inst.overload_set_id, SemIR::SpecificId::None);
+
+  ReplaceInstBeforeConstantUse(context, overload_set_inst_id,
+                               overload_set_inst);
+
+  context.imports().push_back(overload_set_inst_id);
+  return overload_set_inst_id;
+}
+
+// Gets the access for an overloaded function set. Returns std::nullopt
+// if the access is not the same for all functions in the overload set.
+// TODO: Fix to support functions with different access levels.
+static auto GetOverloadSetAccess(Context& context, SemIR::LocId loc_id,
+                                 const clang::UnresolvedSet<4>& overload_set)
+    -> std::optional<SemIR::AccessKind> {
+  auto access = overload_set.begin().getAccess();
+  for (auto it = overload_set.begin(); it != overload_set.end(); ++it) {
+    if (it.getAccess() != access) {
```

Comment:
The first iteration is not necessary (checking `overload_set.begin().getAccess() != overload_set.begin().getAccess()`).

You can solve this in different ways. For example:

```suggestion
  auto it = overload_set.begin()
  auto access = it.getAccess();
  ++it;
  for (; it != overload_set.end(); ++it) {
    if (it.getAccess() != access) {
```

```suggestion
  auto access = overload_set.begin().getAccess();
  for (auto it = overload_set.begin() + 1; it != overload_set.end(); ++it) {
    if (it.getAccess() != access) {
```

Reply:
Done.